compiler/rustc_codegen_gcc/src/builder.rs

```diff
@@ -1226,6 +1226,10 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
         (value1, value2)
     }
 
+    fn filter_landing_pad(&mut self, pers_fn: RValue<'gcc>) -> (RValue<'gcc>, RValue<'gcc>) {
+        self.cleanup_landing_pad(pers_fn)
```

Comment:
This seems incorrect?

Reply:
The proper way to encode this in GCC would be `<<<eh_filter (NULL)>>>` (or even better, `<<<eh_must_not_throw (terminate)>>>`, which is something we hoped for for LLVM!). But I don't think there is support in libgccjit yet, @antoyo to confirm?

Reply:
Yeah, I'll take care of this. I'm not done with supporting unwinding yet.

But, please add a `// TODO(antoyo): generate the correct landing pad`.

For my personal information, what are `<<<eh_filter (NULL)>>>` and `<<<eh_must_not_throw (terminate)>>>`? I've never seen this syntax. Are they C attributes?

I don't remember exactly what `filter` is doing, but the proper implementation of this method might be very similar as `cleanup_landing_pad`, but without adding to `self.cleanup_blocks`.

Reply:
These are part of generic/gimple: https://godbolt.org/z/5adjbahnd